.gitmodules

```
@@ -1,28 +1,33 @@
-[submodule "sorc/ufs_model.fd"]
-  path = sorc/ufs_model.fd
-  url = https://github.com/ufs-community/ufs-weather-model
-  ignore = dirty
 [submodule "sorc/wxflow"]
   path = sorc/wxflow
   url = https://github.com/NOAA-EMC/wxflow
 [submodule "sorc/gfs_utils.fd"]
-  path = sorc/gfs_utils.fd
-  url = https://github.com/NOAA-EMC/gfs-utils
+	path = sorc/gfs_utils.fd
+	url = https://github.com/HenryWinterbottom-NOAA/gfs-utils
+	branch = feature/gwdev_issue_2329
 [submodule "sorc/ufs_utils.fd"]
-  path = sorc/ufs_utils.fd
-  url = https://github.com/ufs-community/UFS_UTILS.git
-[submodule "sorc/verif-global.fd"]
-  path = sorc/verif-global.fd
-  url = https://github.com/NOAA-EMC/EMC_verif-global.git
+	path = sorc/ufs_utils.fd
+	url = https://github.com/HenryWinterbottom-NOAA/UFS_UTILS
+	branch = feature/gwdev_issue_2329
+[submodule "sorc/gsi_utils.fd"]
+	path = sorc/gsi_utils.fd
+	url = https://github.com/HenryWinterbottom-NOAA/GSI-utils
+	branch = feature/gwdev_issue_2329
 [submodule "sorc/gsi_enkf.fd"]
-  path = sorc/gsi_enkf.fd
-  url = https://github.com/NOAA-EMC/GSI.git
+	path = sorc/gsi_enkf.fd
+	url = https://github.com/HenryWinterbottom-NOAA/GSI
+	branch = feature/gwdev_issue_2329
```

GSI has been updated to Rocky 8
```suggestion
  path = sorc/gsi_enkf.fd
  url = https://github.com/NOAA-EMC/GSI.git

```